components/server/src/api/configuration-service-api.ts

```diff
@@ -78,9 +78,11 @@ export class ConfigurationServiceAPI implements ServiceImpl<typeof Configuration
         }
 
         const limit = req.pagination?.pageSize || 25;
```

Comment:
Should we have max page size and return invalid argument if a user requests more?

Reply:
For sure, I'll add this in a followup, need to do some work around pagination/cursors anyways.

Reply:
yeah, sorry for the confusion in this area yet, i hope we can settle it today

Reply:
I don't think it is blocking since it is under FF, we can resolve with follow up PR?